style.css

```diff
@@ -19,14 +19,14 @@
 .main {color: aliceblue}
 
 .link {
-    color: aliceblue;
+    color: aliceblue; 
 
-    background: antiquewhite;
-  color: aqua;
+    background: antiquewhite; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/test_test/pullRequest?prid=5457187)